Frontend/src/services/pointServices.js

```diff
@@ -59,7 +59,53 @@ function getPoint(pointId, callback){
     return null;
 };
 
+function updatePoint(point){
+    let content = {
+        'headers':{
+            'token': localStorage.getItem('token')
+        },
+        'body': point
+    }
+    let urlString = 'https://api.pointmap.net/api/point/'+point.Id

```

Comment:
use the api_url constant, import it from the file

Reply:
Nice idea